azure-quantum/environment.yml

```diff
@@ -17,8 +17,8 @@ dependencies:
     - azure-core==1.19.1 # see https://github.com/microsoft/qdk-python/issues/220
     - azure-devtools>=1.2.0
     - azure-storage-blob==12.11.0 # see https://github.com/microsoft/qdk-python/issues/220
-    - cirq-core>=0.13.1
-    - cirq-ionq>=0.13.1
+    - cirq-core>=0.15.0
```

Comment:
I think this one in 0.15.0 is the correct, as that is the version we want to test with